### ISSUE TITLE
[ACPI] Syntax question  while reading SSDT-EC generated by SSDTTime

### ISSUE THREAD

**Issue by 1alessandro1** [dortania/bugtracker#55]
I was wondering if there was a difference between `\_SB.PCI0.EC0` and `_SB.PCI0.EC0`, from the ACPI spec that doesn't seem to be clear. 

Thanks in advance

**Comment by dhinakg** [comment]
`\` specifically means the root. The only reason I could think of this being useful is if you have a name defined at the root scope or parent scope and redefined in a child scope (so for example, `\ABCD` and `\_SB.ABCD`) and you are trying to access the one in the parent scope. For example, in the below SSDT, we are trying to set `\ABCD` to 3:

```c
DefinitionBlock ("", "SSDT", 2, "TEST", "TEST", 0x3)
{
    Name (ABCD, 0x87)
    
    Scope (_SB) {
        Device (ABCD) {
            // Blank for test purposes
        }
        ABCD = 3
    }
}
```
However, this won't compile as we get the following error: `9, 6058, Invalid type ([Device] found, Store operator requires [Integer|String|Buffer|Package|DebugObject|Reference])`, which means the compiler thought we were trying to set `\_SB.ABCD` to `3`, and you can't set a `Device` to an integer.
If we change the line to `\ABCD = 3`, it clears up that we are intending to set the value to the `\ABCD` reference.

Generally, this doesn't occur in ACPI anyway, so you shouldn't have to worry about it.

tl;dr: For most intents and purposes, it doesn't matter.

**Comment by 1alessandro1** [comment]
Thanks a lot for your patience and for such detailed infos... Have a nice day!

**Comment by frozenzero123** [comment]
> 非常感谢您的耐心配合以及如此详细的信息...祝您有美好的一天！

hello did you have any update? We have the same machine, Yogas740. Now we have driven sound cards and so on. We have done everything that can drive them, but there is still no way for the graphics card. Do you have any good progress? Stuck in panic unsupported CD Clock.......
![InkedQQ图片20200811091511_LI](https://user-images.githubusercontent.com/52648473/89865849-0d91b980-dbe1-11ea-87a9-265247bbc4a2.jpg)
